src/main/java/testapp/adapter/memory/InMemoryUserRepository.java

```diff
@@ -26,9 +25,9 @@ public InMemoryUserRepository() {
   }
 
   @Override
-  public User findUser(Integer userId) {
+  public User findUser(Integer facultyNumber) {
     for (User user : userList) {
-      if (user.getId().equals(userId)) {
+      if (user.getFacultyNumber().equals(facultyNumber)) {
```

Comment:
I have to ask, are we certain that the facultyNumber is so unique as to use it for the finding and deletion of users?
Also what is the value of the facultyNumber to this app?